cli/linter/schema.json

```diff
@@ -1023,6 +1023,18 @@
     },
     "ssl_force_common_name_check":{
 	    "type":"boolean"
+    },
```

Comment:
We already have an existing health check endpoint, see `HealthCheckEndpointName`. I do not like introducing new option here, and rather will think how to make it backward compatible. Any thoughts on it?

Regarding separate port, `HealthCheckEndpointName` we have now already running on ControlAPI router, so if you move it to the separate port/domain, health check gets moved too.